PositionalGuide/ConfigWindow.cs

```diff
@@ -232,7 +269,13 @@ public class ConfigWindow: Window, IDisposable {
 			ImGui.TextUnformatted("");
 			ImGui.TextUnformatted("Target may be any of the following, hyphyens optional, to toggle that line:");
 			ImGui.Indent();
-			ImGui.TextUnformatted("fl, front-left, f, front, fr, front-right, r, right, br, back-right, b, back, bl, back-left, l, left, c, circle");
+			ImGui.TextUnformatted("fl, front-left, f, front, fr, front-right, r, right, br, back-right, b, back, bl, back-left, l, left");
+			ImGui.Unindent();
+			ImGui.TextUnformatted("Targets for circles are:");
+			ImGui.Indent();
+			ImGui.TextUnformatted("For the target circle: c, circle, circle_target");
+			ImGui.TextUnformatted("For the outer circle: co, circle_outer, outer");
```

Comment:
Likewise here, change `_` to `-` and maybe swap it to be `outer-circle` instead.